src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/VeteranInformationComponent.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+const VeteranInformationComponent = props => {
+  let dateOfBirthFormatted = '';
+  let genderFull = '';
+  if (props.user && props.user.dob) {
+    dateOfBirthFormatted = moment(props.user.dob).format('MMMM Do YYYY');
+  }
+  if (props.user && props.user.gender === 'M') {
+    genderFull = 'Male';
+  } else if (props.user && props.user.gender === 'F') {
+    genderFull = 'Female';
+  }
+  const alertContent = (
+    <>
+      <p>
+        <strong>
+          {props.user.userFullName.first} {props.user.userFullName.last}
+        </strong>
+      </p>
+      <p>Date of birth: {dateOfBirthFormatted}</p>
+      <p>Gender: {genderFull}</p>
```

Comment:
Should these be wrapped in some sort of check? Something like `{dateOfBirthFormatted ? (<p>...</p> : null}`? I know the hope is that since we're pulling from the user object hydrated at login this information should most likely always be available, but it might be worthwhile to conditionally render this anyway.

Reply:
That's why I have the checks for dob and for gender as well as setting them to empty strings before the checks, if the data isn't there then they value just remains an empty string.